sections/futures/UserInfo/OpenOrdersTable.tsx

```diff
@@ -84,7 +102,14 @@ const OpenOrdersTable: React.FC = () => {
 						);
 					},
 					onExecute: () => {
-						dispatch(executeDelayedOrder(o.marketAddress));
+						if (!o.marketKey) return null;
```

Comment:
Not really related to this PR but is there a reason for marketKey to be nullable? Seems we should never have an order with no market

Reply:
Fixed this with a comment above. It's because we don't want to pass the entire market to the SDK query, so I added a few nullable fields that get hydrated when the orders come back. I decided to just make them required. It would only exclude orders where we don't have markets, which isn't actually possible.